practice/src/js/products/productForm.js

```diff
@@ -75,41 +75,35 @@ export default class ProductForm {
 
       const product = formData.product;
 
-      try {
-        let response;
+      let response;
 
-        if (this.action === ACTIONS.ADD) {
-          response = await this.productService.add(product);
-        } else if (this.action === ACTIONS.EDIT) {
-          const productId = new URLSearchParams(window.location.search).get(
-            'id'
-          );
+      if (this.action === ACTIONS.ADD) {
+        response = await this.productService.add(product);
+      } else if (this.action === ACTIONS.EDIT) {
```

Comment:
@JennyDoJD Please reduce the use of if-else statements in your code. This will help make your code clearer and more readable